package.json

```
@@ -37,12 +36,13 @@
     "lib": "./lib"
   },
   "devDependencies": {
-    "eslint": "^7.13.0",
-    "eslint-config-digitalbazaar": "^2.8.0",
-    "eslint-plugin-jsdoc": "^37.9.7",
-    "jsdoc-to-markdown": "^7.1.1"
+    "eslint": "^8.57.1",
+    "eslint-config-digitalbazaar": "^5.2.0",
+    "eslint-plugin-jsdoc": "^50.6.3",
+    "eslint-plugin-unicorn": "^55.0.0",
+    "jsdoc-to-markdown": "^9.1.1"
```

remove unused dependency
```suggestion
    "eslint-plugin-unicorn": "^55.0.0"
```